api/envoy/config/core/v3/base.proto

```diff
@@ -165,6 +167,10 @@ message Node {
   // directly to the management server.
   google.protobuf.Struct metadata = 3;
 
+  // Map from xDS resource type URL to dynamic context parameters. These may
+  // vary at runtime (unlike other fields in this message).
```

Comment:
What do you mean by vary at runtime? You mean because some context params are pulled from the environment? Is this documented anywhere that we can link to?

Reply:
I've extended with an example. In Envoy, this is modified at runtime on the `Server::Instance` interface, but that is fairly client specific. Eventually we will have the dynamic context parameter proposal that @markdroth is working on that will be another source of these dynamic parameters.

Reply:
FWIW, I'm actually open to moving this to `DiscoveryRequest` from `Node`; my original motivation for not doing this is that I didn't want to complicated the transport protocol with a new field that is really there to serve a legacy use case, i.e. dynamic parameters are mostly intended for forward facing use cases built on `xdstp://`, but we add them to `Node` as a legacy workaround when needed (as is the case with some of our use cases).

Reply:
I don't feel strongly about it either way.